fu/basic/Fu.py

```diff
@@ -84,11 +84,13 @@ def proceed_latency():
 
     @update
     def update_reached_vector_factor():
-      s.reached_vector_factor @= 0
+      #s.reached_vector_factor @= 0
       if s.recv_opt.val & (s.vector_factor_counter + \
                            (VectorFactorType(1) << zext(s.vector_factor_power, VectorFactorType)) >= \
                            (VectorFactorType(1) << zext(s.recv_opt.msg.vector_factor_power, VectorFactorType))):
         s.reached_vector_factor @= 1
+      else:
+        s.reached_vector_factor @= 0
```

Comment:
During your previous synthesis result, was there latch related to `reached_vector_factor`? Based on @yo96, as long as there is `s.reached_vector_factor @= 0` at line 87, we don't need this `else`, and there won't be latch.